tripleo/insights/git.py

```diff
@@ -0,0 +1,138 @@
+"""
```

Comment:
This feels very generic git module, even overlaps with what we have in Cibyl. Don't you want to just import from cibyl and consolidate git-related code in one place?

Reply:
I am approaching the library as its own independent thing, so that it does not depend on Cibyl to work. Some of the utilities do overlap between the two sources though. I could either make the library depend on Cibyl or create a 'utils' folder on the project's root where shared sources are added.